pkg/kudoctl/cmd/client/client.go

```diff
@@ -0,0 +1,35 @@
+package client
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	kudoinit "github.com/kudobuilder/kudo/pkg/kudoctl/cmd/init"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/env"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/kube"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+)
+
+// GetClient is a helper function that takes the Settings struct and returns a new KUDO Client
```

Comment:
Let's add more docs on what the validation actually mean

Reply:
`GetClient` does not match the func `GetValidatedClient` .  
Still don't like "ValidatedClient`.   It isn't clear what it is communicating.